### PR TITLE
Implement brightness control and 'Leave Game' button, add another Socket option

### LINKED ISSUE #326: Implement brightness control
**Is your feature request related to a problem? Please describe.**
The brightness control in the setting is pseudo

**Describe the solution you'd like**
Change the background colour to adjust the brightness

**Additional context**
Linked Issue #265

**Location**
Create game -> Setting

### Feature Approval:
- [x]  Feature approved by:

### LINKED ISSUE #327: Implement 'Leave Game' Button
**Is your feature request related to a problem? Please describe.**
The 'Leave Game' button in the setting is pseudo

**Describe the solution you'd like**
Reload the window, other gamers will see an inactive gamer who always does nothing till time out.

**Describe alternatives you've considered**
Send a signal to self-destruct, and others will see this player's status to be 'DEAD' immediately.

**Additional context**
Linked Issue #265

**Location**
Create game -> Setting

### Feature Approval:
- [x]  Feature approved by:

### PR DESCRIPTION
**Pull request by ysway**
# Description
This PR fixes #326
  - By changing the background colour to adjust the brightness

This PR fixes #327
  - Return to the home page when leaving the game

# Demo
image
image

# Type of change:
* New feature (Non-breaking change that adds functionality)

# Additional context
Partially implemented BGM and dummy voting time and discussion time settings

### CODE REVIEW

File: frontend/puppeteer/package-lock.json [hunk 1]
```diff
@@ -1,8 +1,488 @@
 {
     "name": "puppeteer-debug-tool-for-mafia",
```
Comment (ODE-BOY) [inline frontend/puppeteer/package-lock.json:2]: Is this puppeteer file necessary to the issue? Was it just created by the npm install instruction, and do we want it to be added to the repo?

**Review by ODE-BOY — DISMISSED** [review]
Looks good to me. Thanks for doing this.
Im not completly sure if we want to commit the package-lock.json files to the repo but part from that I think its good to go.

**Comment by ysway** [comment]
> Looks good to me. Thanks for doing this.
> Im not completly sure if we want to commit the package-lock.json files to the repo but part from that I think its good to go.

It is necessary for passing the GitHub action auto-check, since the action uses ```npm ci```, an updated package-lock.json file will be needed if there are new imported packages.

File: frontend/puppeteer/package-lock.json (hunk 1 above)
Comment (ysway) [reply frontend/puppeteer/package-lock.json:2]: I can't pass the husky lint check of commit if the import is missing, so an updated file is necessary.
But the file is not necessary for the issue

**Review by maheknangia — APPROVED** [review]
Looks good to me! Thanks for this Siwei!!

**Review by ODE-BOY — APPROVED** [review]
LGTM

**Pull request merged**